config/routes.rb

```diff
@@ -10,7 +10,7 @@
   get 'status/health', to: 'status#health'
   get 'status/readiness', to: 'status#readiness'
 
-  scope "(:locale)", locale: /en|de|fr|zh|ru|ch_be/ do
+  scope "/" do
```

Comment:
Is this even needed now?
Can't you just remove the scope completely? Feels like that should have the same effect

Reply:
Yes, you're right. It is completely unnecessary.